plugins/module_utils/mh/mixins/cmd.py

```diff
@@ -88,6 +88,7 @@ class CmdMixin(object):
     """
     command = None
     command_args_formats = {}
+    command_output_as_result = True
```

Comment:
I don't think this should be a global option, but (also) a parameter to `run_command`.

Also there are cases (such as for `snap`) where being able to just not add `rc` could be useful. Having `stdout` and `stderr` from the last executed command can be useful for troubleshooting.

Reply:
Yeah, it makes more sense to make it per-call to run_command.